src/scripts/i18n/list_source_files.py

```
@@ -0,0 +1,113 @@
+
+SUFFIX_MAP: Dict[str, List[str]] = {
+    "components": [".component.ts", ".component.html"],
+    "services": [".service.ts"],
+    "pipes": [".pipe.ts"],
+    "directives": [".directive.ts"],
+}
+
+ALL_TYPES = ",".join(SUFFIX_MAP.keys())
+
+
+def find_files(root_dir: str, suffixes: List[str]) -> List[str]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)